### DISCUSSION TITLE
How to encode Dataset BY-License attribution?

### DISCUSSION THREAD

**Discussion by hbruch** [schemaorg/schemaorg#2791] [Q&A] [answered]
I'd like to publish schema data for datasets. Some of these are CC-BY licensed and I'd like to add the appropriate attribution to the dataset description. How should this be represented?

**Comment by hbruch** [comment, marked as answer]
Ok, I think [creditText](https://schema.org/creditText) would be the property to use

**Comment by thadguidry** [comment]
Your "attribution" use case might be a better fit for just using the [publisher](https://schema.org/publisher), [provider](https://schema.org/provider), [producer](https://schema.org/producer), or simply [author](https://schema.org/author) or [creator](https://schema.org/creator) and sometimes the "legally accountable person" with [accountablePerson](https://schema.org/accountablePerson)

But if you just want to use a quick and dirty [creditText](https://schema.org/creditText) that's fine, if indeed it's some dataset citation that has a credit or attribution string...but you could always make this better with a "Thing instead of a String" with good structure if you know it's a [provider](https://schema.org/provider) of the dataset for example then you would want to use that more specific property.

This depends on the context, since "attribution" means different things in different contexts.

Also if you didn't know, any property in Schema.org can simply use Text even though that's not readily self-evident, and doesn't have to necessarily use a Type.
